pymc_extras/inference/laplace_approx/find_map.py

```diff
@@ -336,7 +336,7 @@ def find_MAP(
     }
 
     idata = map_results_to_inference_data(optimized_point, frozen_model, include_transformed)
-    idata = add_fit_to_inference_data(idata, raveled_optimized, H_inv)
+    idata = add_fit_to_inference_data(idata, raveled_optimized, H_inv, model)
```

Comment:
To avoid name confusion?

```suggestion
    idata = add_fit_to_inference_data(idata, raveled_optimized, H_inv, model=model)
```